src/pages/ConvertVrt/Convert/index.tsx

```diff
@@ -0,0 +1,122 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import BigNumber from 'bignumber.js';
+import noop from 'noop-ts';
+import { Typography } from '@mui/material';
+import {
+  FormikTokenTextField,
+  LabeledProgressBar,
+  FormikSubmitButton,
+  ConnectWallet,
+  Icon,
+} from 'components';
+import { useTranslation } from 'translation';
+import useConvertToReadableCoinString from 'utilities/useConvertToReadableCoinString';
+import { format } from 'utilities/common';
+import { ConvertVrtForm } from '../Form';
+import { VRT_ID, XVS_ID } from '../constants';
+import { useStyles } from '../styles';
+
+interface ConvertProps {
+  xvsTotal: BigNumber;
+  xvsToVrtRate: BigNumber;
+  vrtLimitUsed: number;
+  vrtLimit: number;
+  vestingTime: number;
+}
+
+const Convert: React.FC<ConvertProps> = ({
+  xvsTotal,
+  xvsToVrtRate,
+  vrtLimitUsed,
+  vrtLimit,
+  vestingTime,
+}) => {
+  const styles = useStyles();
+  const { t, Trans } = useTranslation();
+  const readableXvsAvailable = useConvertToReadableCoinString({
+    valueWei: xvsTotal,
+    tokenId: XVS_ID,
+  });
+  return (
+    <div css={styles.root}>
+      <section css={styles.title}>
+        <Typography variant="h3">{readableXvsAvailable}</Typography>
+        <Typography variant="small2">{t('convertVrt.xvsAVailable')}</Typography>
+      </section>
+      <ConnectWallet message={t('convertVrt.connectWalletToConvertVrtToXvs')}>
+        <ConvertVrtForm
+          onSubmit={noop}
+          maxVrt={new BigNumber(vrtLimit).toFixed()}
+          maxXvs={new BigNumber(xvsTotal).toFixed()}
+        >
+          {() => (
+            <>
+              <div css={styles.inputSection}>
+                <Typography variant="small2" css={styles.inputLabel}>
+                  {t('convertVrt.convertVrt')}
+                </Typography>
+                <FormikTokenTextField
+                  tokenId={VRT_ID}
+                  name="vrt"
+                  max={xvsTotal.toFixed()}
```

Comment:
I think this line should be removed and instead we should make the border of the input red if the value is higher than the max, because users are also able to change the value of the XVS input which means they might enter a value that's higher than the max allowed